Sources/AppAuth/macOS/OKTRedirectHTTPHandler.m

```diff
@@ -127,11 +127,22 @@ - (void)stopHTTPListener {
   _httpServ = nil;
 }
 
+- (BOOL)isOptionsHTTPServerRequest:(HTTPServerRequest *)request {
+    CFStringRef method = CFHTTPMessageCopyRequestMethod(request.request);
+    if (method == nil) {
+        return NO;
+    }
+
+    BOOL isOptionsRequest = CFStringCompare(method, (__bridge CFStringRef)@"OPTIONS", 0) == kCFCompareEqualTo;
```

Comment:
I'd suggest using `kCFCompareCaseInsensitive` option for CFStringCompare (just like `CFHTTPMessageCopyRequestMethod` does internally for `GET` method).

Reply:
That makes sense. Wanted to introduce minimal changes to the existing logic though. But we can cover this in unit tests, lemme do this the change

Reply:
Addressed. Added more tests